src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -144,17 +146,20 @@ def execute(self, reqStatus):
 
         # Call _execute() and feed the relevant pipeline with the objects popped from requestRecords
         try:
-            totalNumRequests, cleanNumRequests, archivedNumRequests = self._execute(requestRecords)
+            totalNumRequests, cleanNumRequests, normalArchivedNumRequests, forceArchivedNumRequests = self._execute(requestRecords)
```

Comment:
I wonder why this method returns 
```
cleanNumRequests, normalArchivedNumRequests, forceArchivedNumRequests
```

if there is already an object attribute counter for that? Shouldn't it simply return `totalNumRequests` and the rest of the information is available in `self.wfCounters`?

Reply:
At the beginning I did it for consistency of how this was done before and then I found it quite useful in the unit tests to have all what I need returned in a tuple here. Because there we do not call the object from `MSManager`, but we rather create an instance for `MSRulecleaner` and we deal with it directly. Not saying it is impossible for the unit test to be fixed too, in case we change the returned value here though, but it will for sure require more than a change of a single number in the tuple, as I did for fixing it this time.